modular_citadel/code/modules/reagents/chemistry/reagents/MKUltra.dm

```diff
@@ -0,0 +1,373 @@
+/*
+////////////////////////////////////////
+//				MKULTA				  //
+////////////////////////////////////////
+The magnum opus of FermiChem -
+Long and complicated, I highly recomend you look at the two other files heavily involved in this
+modular_citadel/code/datums/status_effects/chems.dm - handles the subject's reactions
+code/modules/surgery/organs/vocal_cords.dm - handles the enchanter speaking
+
```

Comment:
@Shadeykins @CornMyCob I want your opinions on whether we even want this. Its extremely overpowered and pushes heavy vibes for content we dont want here. Up to you

Reply:
_Having domination chems reference government sponsored torture is my fetish._

Big nope.